app/AppKernel.php

```diff
@@ -19,6 +19,7 @@ public function registerBundles()
             new FOS\UserBundle\FOSUserBundle(),
             new JavierEguiluz\Bundle\EasyAdminBundle\EasyAdminBundle(),
             new PlantBundle\PlantBundle(),
+            new \ChlorisEasyAdminBundle\ChlorisEasyAdminBundle(),
```

Comment:
name of this bundle should be exact the same: `EasyAdminBundle`

Otherwise it will not override